<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SinglePlayer</class>
 <widget class="QWidget" name="SinglePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MyPushButton" name="menuButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Chinese Rocks Rg</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/menu/img/ButtonFrame.png);
background-image: url(:/menu/img/ALLGAMEBG.png);
background-position: center;
background-repeat: no-repeat;
background-attachment: fixed;

color: rgb(164, 175, 141);</string>
       </property>
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pauseButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Chinese Rocks Rg</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/menu/img/ButtonFrame.png);
background-image: url(:/menu/img/ALLGAMEBG.png);
background-position: center;
background-repeat: no-repeat;
background-attachment: fixed;

color: rgb(164, 175, 141);</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="resetButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Chinese Rocks Rg</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/menu/img/ButtonFrame.png);
background-image: url(:/menu/img/ALLGAMEBG.png);
background-position: center;
background-repeat: no-repeat;
background-attachment: fixed;

color: rgb(164, 175, 141);</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="ScoreLabel">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>22</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(164, 175, 141);</string>
         </property>
         <property name="text">
          <string>SCORE:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="PointsLabel">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(164, 175, 141);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="interactive">
      <bool>true</bool>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>600.000000000000000</width>
       <height>400.000000000000000</height>
      </rectf>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
